Scripts/lsp.js

```diff
@@ -2,7 +2,7 @@ class RubyLanguageServer {
   constructor() {
     // Observe the configuration setting for the server's location, and restart the server on change
     nova.config.observe(
-      "ruby.language-server-path",
+      "com.roguelazer.RubyLsp.languageServerPath",
```

Comment:
This is technically a breaking change. However, this key is the same [key used in the original extension](https://github.com/tdegrunt/ruby.novaextension/blob/main/Scripts/lsp.js#L4) so it's already overloaded; changing one will (I believe) change the other. Given that there are 22 possible users of this extension (at the time of writing), I thought it might be best to break that now.